Cargo.lock

```diff
@@ -2743,15 +2744,30 @@ dependencies = [
  "wasmtime-types",
 ]
 
+[[package]]
+name = "crc"
+version = "2.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "49fc9a695bca7f35f5f4c15cddc84415f66a74ea78eef08e90c5024f2b540e23"
+dependencies = [
+ "crc-catalog 1.1.1",
+]
```

Comment:
We will get rid of this old dep in the next encointer-release aka for polkadot-v1.13.0, see https://github.com/encointer/pallets/pull/396